src/Docker/deploy.msbuild

```diff
@@ -1,301 +1,43 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Deploy"  ToolsVersion="4.0">
+
 	<PropertyGroup>
-		<GXDeployFileProject>$([System.IO.Path]::GetFullPath('$(DeployFullPath)\..\..\..\..\web\$(ProjectName).gxdproj'))</GXDeployFileProject>
-		<MainPrefix>a</MainPrefix>
+		<DeployDirectory>$([System.IO.Path]::GetFullPath('$(DeployFullPath)\..\..\context'))</DeployDirectory>
+		<Dockerfile>$(DeployDirectory)\Dockerfile</Dockerfile>
 	</PropertyGroup>
-	<Import Project="$(GX_PROGRAM_DIR)\GeneXus.Docker.targets"/>
-	<Import Project="$(WebSourcePath)\$(DeploymentUnit).props" Condition="Exists('$(WebSourcePath)\$(DeploymentUnit).props')"/>
-	<Import Project="$(GXDeployFileProject)" Condition ="Exists('$(GXDeployFileProject)')" />
-	<Import Project="$(GX_PROGRAM_DIR)\Imports.targets" Condition ="!Exists('$(GXDeployFileProject)')"/>
-
-	<Target Name="CreateDockerfile">		
-		<GetDockerConfigDefaults Condition="'$(DOCKER_WEBAPPLOCATION)' == '' OR '$(DOCKER_BASE_IMAGE)' == ''"
-			BuildEnvironment = "$(BuildEnvironment)"
-			Generator = "$(GENERATOR)" 
-			IsCMD = "$(IsCMD)" >
-			<Output TaskParameter="WebAppLocation"
-					PropertyName="DOCKER_WEBAPPLOCATION" />
-			<Output TaskParameter="BaseImage"
-					PropertyName="DOCKER_BASE_IMAGE" />
-		</GetDockerConfigDefaults>
-
-		<ItemGroup>
-			<SourceFile Include="$(DeploySource)" />
-		</ItemGroup>
-
-		<Error Text="Property 'Image App location' cannot be empty" Condition="'$(DOCKER_WEBAPPLOCATION)' == ''"/>
-
-		<Error Text="DeploySource cannot be empty" Condition="'$(DeploySource)' == ''"/>
-
-		<Error Text="DeploySource must be a .war or .jar file" Condition="'$(GENERATOR)' == 'Java' AND '%(SourceFile.Extension)' != '.war' AND '%(SourceFile.Extension)' != '.jar'" />
-		<Error Text="DeploySource must be a .zip file" Condition="('$(GENERATOR)' == 'C#' OR '$(GENERATOR)' == '.NET Framework' OR '$(GENERATOR)' == '.NET Core' or '$(GENERATOR)' == '.NET') AND '%(SourceFile.Extension)' != '.zip'"/>
 
-		<Error Text="The deployment unit must have only one command line procedure." Condition="'@(SelectedObject->Count())' &gt; 1 AND '$(IsCMD)' == 'true'"/>
+	<Import Project= "createpackage.msbuild" Condition = "'$(CreateCloudPackage)' == '' or '$(CreateCloudPackage)' == 'true'" />
```

Comment:
The "CreateCloudPackage" is for compatibility with preexisting Msbuild deploy scripts, right?

Reply:
yes, exactly